apiserver/plane/app/views/issue/custom_property.py

```
@@ -33,15 +34,62 @@ class IssueCustomPropertyUpdateAPIView(BaseAPIView):
+                except (ValueError, TypeError):
+                    return custom_property, Response(
+                        {"error": f"Invalid date format. Date must be in YYYY-MM-DD format (e.g., 2025-04-24)."},
+                        status=status.HTTP_400_BAD_REQUEST
+                    )
+            case _:  # Default case for text type
+                # For text type, we just use the value field directly
+                pass
+        
+        return custom_property, None
+    
     def patch(self, request, slug, issue_id, pk):
```

Pylint (3.3.7)

[refactor] Too many local variables (17/15)

(R0914)